core/embed/sys/powerctl/stm32u5/powerctl_suspend.c

```diff
@@ -93,6 +95,10 @@ void powerctl_suspend(void) {
       // immediately after exiting STOP2 mode.
       irq_key_t irq_key = irq_lock();
 
+      // Enable PWR peripheral clock
+      // (required by the following HAL_PWREx_EnterSTOP2Mode)
+      __HAL_RCC_PWR_CLK_ENABLE();
```

Comment:
maybe we should disable after waking up? (it is enabled again in SystemInit and then disabled, so end effect zero, but it is not very clear from this function here)